### ISSUE TITLE
the README should be updated to mention CSP implications

### ISSUE THREAD

**Issue by ghost** [xomaczar/ember-cli-styles-reloader#7]
By default, the animate changes code would cause a CSP warning in browsers that support it, due to the inline style element.

**Comment by xomaczar** [comment]
Agree.
Do you mind making a PR with an sample CSP configuration.

**Comment by ghost** [comment]
I don't think that's a good idea to add the current code to their CSP configuration as that would mean enabling all inline styles. We have at least these two options:
1. make the inline style a class on a div (or whatever element) and suggest people add the css  to their app.
2. OR set a script nonce/hash on style element itself and then suggest people add that nonce/hash to their CSP config.

(2) would be possible to implement automatically by appending to the active CSP configuration, but I'm not really sure of the browser support.

**Comment by xomaczar** [comment]
Do you have time to work on this. I am currently slammed.

**Comment by ghost** [comment]
you didn't mention which way you wanted to go, or even if you agreed with my assessment.

**Comment by xomaczar** [comment]
How about instead of inline css, we import animation style into app.css in dev environment.

**Comment by ghost** [comment]
import it how? Honestly I still haven't done much with ember at all. I'm way more familiar with the internals of ember-cli than I am with actually using it :)

**Comment by xomaczar** [comment]
I meant we can simply merge it to app/style or vendor tree, either one should work. This should silence the CSP.

``` javascript
 included: function(app) {
    this._super.included(app);
    //check if development only
    app.import('animation-reloader-something.css');
  }
```

**Comment by xomaczar** [comment]
Fixed via #11
